.github/workflows/coding-standards.yml

```
@@ -0,0 +1,119 @@
+name: Coding Standards
+
+on:
+  push:
```

```suggestion
  push:
    branches:
      - master
```

The `pull_request` event will catch any updates to the PR including pushes to that branch, so here we can limit to the `master` branch and probably anything in the `*.*` format for backport purposes.

---

I made this change with a small tweak. The coding standards scanning was not introduced until 5.1. So I added two patterns, one matching `5.[1-9]`, and one matching `[6-9]-*`. That should match all branches since 5.1 and going forward (until 10.0).